### PR TITLE
Bugfix/#724

### LINKED ISSUE #724: Send an email when a user changes the password
### Steps to reproduce the error
1. Click on user name > Edit profile.
2. Click on Change Password.
3. Change the password.
4. Check your inbox.
### Actual behavior

No email is sent to inform about the password change.
### Expected behavior

An email to inform about the password change should be sent

### PR DESCRIPTION
**Pull request by aklife97**
### Proposed changes in this pull request

- Changes /cadasta/accounts/models.py
- Fixes #724 

### When should this PR be merged

- Works fine. Can be merged

### Risks

- None

### Follow up actions

- None

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description. 
- [ ] **Is the PR labeled correctly?** It should have the `migration` label if a new migration is added. 
- [ ] **Is the risk level assessment sufficient?** The risks section should contain all risks that might be introduced with the PR and which actions we need to take to mitigate these risks. Possible risks are database migrations, new libraries that need to be installed or changes to deployment scripts.

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. 

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. 
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). 

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components. 
- [ ] **If this is a bug fix, are tests for the issue in place**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes must be documented in the [Cadasta Platform Documentation](https://github.com/Cadasta/cadasta-docs).
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented in the [API docs](https://github.com/Cadasta/api-docs).
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

**Comment by aklife97** [comment]
@oliverroick Fixed the code to pass the PEP8 violations. Will not forget to lint before sending other pull requests :)

**Comment by aklife97** [comment]
@oliverroick Hi,
I am facing a bit trouble with makemessages script here. Even after executing the script, I don't get to see the translation in the email that is sent. Could you please help me out with this.
Also, should I push only the changes in code and templates or should I also include the newly created .po files.
Thanks

**Comment by oliverroick** [comment]
Don't worry about running `makemessages`; we'll take care of that. Also don't add your `.po` files please; it'll make things easier. 

The reason you don't see a translated message is probably because you haven't provide a translation or the translations are not compiled. That said, you don't need to provide translations, we're using Transifex for that. Only mark it for translation and that's it

**Comment by aklife97** [comment]
Okay, I have updated as per what you asked. Please see to it :)

File: cadasta/accounts/models.py [hunk 1]
```diff
@@ -89,3 +93,18 @@ def assign_default_policy(sender, instance, **kwargs):
...
         assigned_policies.insert(0, policy)
     instance.assign_policies(*assigned_policies)
+
+
+@receiver(password_changed)
+@receiver(password_reset)
+def password_changed_reset(sender, request, user, **kwargs):
+    msg_body = get_template(
+        "accounts/email/password_change_successful.txt"
+        ).render(Context({'user': user}))
```
Comment (oliverroick) [inline cadasta/accounts/models.py]: The context is not used in the template; there's no need to create and pass it.

**Comment by oliverroick** [comment]
This PR has been inactive for two weeks. Closing this in favour of #1174

**Pull request closed without merging**